packages/headless/src/controllers/commerce/facets/core/regular/headless-commerce-regular-facet.test.ts

```diff
@@ -79,26 +79,4 @@ describe('CommerceRegularFacet', () => {
       );
     });
   });
-
-  describe('#toggleSingleSelect', () => {
```

Comment:
No longer needed since we're testing that #toggleSingleSelect and #toggleSingleExclude are calling #toggleSelect and #toggleExclude respectively in the core facet controller.